engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableAggregationTest.java

```diff
@@ -116,12 +116,12 @@ private static Table individualStaticByTest(@NotNull final Table input,
                     Arrays.stream(keySelectColumns).map(SelectColumn::getName).distinct().toArray(String[]::new);
 
             if (keyColumns.length == 0) {
-                expectedKeys = TableTools.emptyTable(adjustedInput.size() > 0 ? 1 : 0);
+                expectedKeys = TableTools.emptyTable(!adjustedInput.isEmpty() ? 1 : 0);
                 expected = adjustedInput;
             } else {
                 final Set<String> retainedColumns =
-                        new LinkedHashSet<>(adjustedInput.getDefinition().getColumnNameMap().keySet());
-                retainedColumns.removeAll(Arrays.stream(keyNames).collect(Collectors.toSet()));
+                        new LinkedHashSet<>(adjustedInput.getDefinition().getColumnNameSet());
+                Arrays.asList(keyNames).forEach(retainedColumns::remove);
```

Comment:
Isn't this better?
```suggestion
                retainedColumns.removeAll(Arrays.asList(keyNames));
```

Reply:
So, IJ warned me and apparently the semantics (as explicitly called out in) `java.util.AbstractSet#removeAll` is *BAD* because it may end up doing contains call against the RHS. It was a big oversight in java development, and they've been unable to fix this.

If RHS is a set `AbstractSet.removeAll` is _ok_.